client/components/thank-you-v2/footer/index.tsx

```diff
@@ -0,0 +1,69 @@
+import { recordTracksEvent } from '@automattic/calypso-analytics';
+import { Button, Gridicon } from '@automattic/components';
+import { localizeUrl } from '@automattic/i18n-utils';
+import { isOutsideCalypso } from 'calypso/lib/url';
+
+import './style.scss';
+
+export type ThankYouDetailProps = {
+	title?: string;
+	description?: string;
+	buttonText?: string;
+	href?: string;
+	clickEventName?: string;
```

Comment:
Given that those three props are specific to the button, I would prefix them to make that clearer:

```suggestion
	buttonText?: string;
	buttonHref?: string;
	buttonClickEventName?: string;
```

Reply:
Good idea. Also noting that we're going to pass a click callback function in favour of passing a track event name, as per https://github.com/Automattic/wp-calypso/pull/86186#discussion_r1447054922